api/services/device_tags.go

```diff
@@ -4,23 +4,25 @@ import (
 	"context"
 
 	"github.com/shellhub-io/shellhub/pkg/models"
-	"github.com/shellhub-io/shellhub/pkg/validator"
 )
 
+// DeviceTags contains the service's function to manage device tags.
 type DeviceTags interface {
-	CreateDeviceTag(ctx context.Context, uid models.UID, name string) error
-	RemoveDeviceTag(ctx context.Context, uid models.UID, name string) error
+	CreateDeviceTag(ctx context.Context, uid models.UID, tag string) error
+	RemoveDeviceTag(ctx context.Context, uid models.UID, tag string) error
 	UpdateDeviceTag(ctx context.Context, uid models.UID, tags []string) error
 }
 
 // DeviceMaxTags is the number of tags that a device can have.
 const DeviceMaxTags = 3
 
-func (s *service) CreateDeviceTag(ctx context.Context, uid models.UID, name string) error {
-	if !validator.ValidateFieldTag(name) {
```

Comment:
That verification isn't more needed because the route already check if the tags are valid.